app/src/sharedTest/java/org/oppia/android/app/parser/ListItemLeadingMarginSpanTest.kt

```diff
@@ -498,15 +498,15 @@ class ListItemLeadingMarginSpanTest {
       Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
     )
     bulletSpan0.drawLeadingMargin(
-      canvas, paint, x, dir, top, 0, bottom, htmlResult,
+      canvas, paint, x, dir, top, 78, bottom, htmlResult,
```

Comment:
Why is the verticality of this rendering different from the bullet item list test above?

Reply:
For Ordered item list I am not computing y position. In Unordered list we have radius based on that y is computed. So there is difference in these tests.